daemon/container.go

```diff
@@ -918,18 +916,6 @@ func (container *Container) Unmount() error {
 	return container.daemon.Unmount(container)
 }
 
-func (container *Container) logPath(name string) (string, error) {
```

Comment:
@LK4D4 that's a design decision. Either we're gonna pass all that path-related logic to individual loggers (b/c some loggers might be using paths) or we're gonna handle special cases in the `daemon` package. 

To me, moving those to the Logger interface and giving the logging driver option to not to use them  sounds like a cleaner solution since `daemon` now does only care about existence of "Logger"s not "Loggers+logging paths".